From c11d92f2044166c10c425887a031961fd91dd699 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sun, 11 Aug 2024 21:17:40 -0700
Subject: [PATCH 2/2] Fix building on Android

Error was:

  nes/nes.hpp:47:11: error: reference to 'uint_t' is ambiguous
    typedef uint_t< 5> uint5;
            ^
---
 nes/nes.hpp | 58 ++++++++++++++++++++++++++---------------------------
 1 file changed, 29 insertions(+), 29 deletions(-)

diff --git a/nes/nes.hpp b/nes/nes.hpp
index ed76f85..a365ff7 100644
--- a/nes/nes.hpp
+++ b/nes/nes.hpp
@@ -40,39 +40,39 @@ namespace NES {
   typedef int32_t int32;
   typedef int64_t int64;
 
-  typedef uint_t< 1> uint1;
-  typedef uint_t< 2> uint2;
-  typedef uint_t< 3> uint3;
-  typedef uint_t< 4> uint4;
-  typedef uint_t< 5> uint5;
-  typedef uint_t< 6> uint6;
-  typedef uint_t< 7> uint7;
+  typedef nall::uint_t< 1> uint1;
+  typedef nall::uint_t< 2> uint2;
+  typedef nall::uint_t< 3> uint3;
+  typedef nall::uint_t< 4> uint4;
+  typedef nall::uint_t< 5> uint5;
+  typedef nall::uint_t< 6> uint6;
+  typedef nall::uint_t< 7> uint7;
   typedef uint8_t    uint8;
 
-  typedef uint_t< 9> uint9;
-  typedef uint_t<10> uint10;
-  typedef uint_t<11> uint11;
-  typedef uint_t<12> uint12;
-  typedef uint_t<13> uint13;
-  typedef uint_t<14> uint14;
-  typedef uint_t<15> uint15;
+  typedef nall::uint_t< 9> uint9;
+  typedef nall::uint_t<10> uint10;
+  typedef nall::uint_t<11> uint11;
+  typedef nall::uint_t<12> uint12;
+  typedef nall::uint_t<13> uint13;
+  typedef nall::uint_t<14> uint14;
+  typedef nall::uint_t<15> uint15;
   typedef uint16_t   uint16;
 
-  typedef uint_t<17> uint17;
-  typedef uint_t<18> uint18;
-  typedef uint_t<19> uint19;
-  typedef uint_t<20> uint20;
-  typedef uint_t<21> uint21;
-  typedef uint_t<22> uint22;
-  typedef uint_t<23> uint23;
-  typedef uint_t<24> uint24;
-  typedef uint_t<25> uint25;
-  typedef uint_t<26> uint26;
-  typedef uint_t<27> uint27;
-  typedef uint_t<28> uint28;
-  typedef uint_t<29> uint29;
-  typedef uint_t<30> uint30;
-  typedef uint_t<31> uint31;
+  typedef nall::uint_t<17> uint17;
+  typedef nall::uint_t<18> uint18;
+  typedef nall::uint_t<19> uint19;
+  typedef nall::uint_t<20> uint20;
+  typedef nall::uint_t<21> uint21;
+  typedef nall::uint_t<22> uint22;
+  typedef nall::uint_t<23> uint23;
+  typedef nall::uint_t<24> uint24;
+  typedef nall::uint_t<25> uint25;
+  typedef nall::uint_t<26> uint26;
+  typedef nall::uint_t<27> uint27;
+  typedef nall::uint_t<28> uint28;
+  typedef nall::uint_t<29> uint29;
+  typedef nall::uint_t<30> uint30;
+  typedef nall::uint_t<31> uint31;
   typedef uint32_t   uint32;
 
   typedef uint64_t   uint64;
-- 
2.34.1

